### PR TITLE
Feat: Draggable Error improvement

### PR DESCRIPTION
**Pull request by akash-groww**
## What does this PR do?
This PR fixes error rate bug. 
This PR also adds support for vertical graph draggable.

## What packages have been affected by this PR?
react-charts

## Types of changes

What types of changes does your code introduce to this project?

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)

- [ ] New feature (non-breaking change which adds functionality)

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist before merging

_Put an `x` in the boxes that apply_

- [x] These changes have been thoroughly tested.

- [ ] Changes need to be immediately published on npm.

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: packages/react-charts/src/LineGraph/index.tsx [hunk 1]
```diff
@@ -107,17 +107,25 @@ const LineGraph = (props: LineGraphProps) => {
...
+
   const getDragColor = (lp: LinePathData) => {
-    const { startIndex, endIndex } = dragPoints ?? {};
+    const { startPoint, endPoint } = dragPoints ?? {};
     const { fill = lp?.color, lineColor, negativeLineColor } = lp?.draggableConfig ?? {};
 
-    if (startIndex === undefined || endIndex === undefined) return lp?.color;
-    const lpStart = lp?.series?.[startIndex]?.[1];
-    const lpEnd = lp?.series?.[endIndex]?.[1];
+    if (startPoint === undefined || endPoint === undefined) return lp?.color;
```
Comment (shubhamgavali70) [inline packages/react-charts/src/LineGraph/index.tsx:121]: You can use isEmpty here

Comment (akash-groww) [reply packages/react-charts/src/LineGraph/index.tsx:121]: Why to use isEmpty if I don't want to have multiple checks except undefined?

**Review by shubhamgavali1998 — APPROVED** [review]

File: packages/react-charts/package.json
Comment (harshitpruthi94) [inline packages/react-charts/package.json:1]: pnpm i and add pnpmlock.yaml file alse

Comment (akash-groww) [reply packages/react-charts/package.json:1]: done

**Review by ghost — APPROVED** [review]

**Review by harshitpruthi94 — APPROVED** [review]

**Pull request merged**